.eslintrc

```diff
@@ -129,12 +129,7 @@
     ],
     "object-shorthand": [2, "properties", { "avoidQuotes": true }],
     "prefer-const": 2,
-    "prettier/prettier": [2, {
-      "bracketSpacing": false,
-      "singleQuote": true,
-      "trailingComma": "es5",
-      "quoteProps": "preserve"
-    }],
+    "prettier/prettier": [2],
```

Comment:
I think you can do away with the square brackets around `2`.

Reply:
Good call.